flang/lib/Lower/PFTBuilder.cpp

```diff
@@ -286,10 +279,20 @@ class PFTBuilder {
     lastLexicalEvaluation = nullptr;
   }
 
+  /// Pop the ModuleLikeUnit evaluationList when entering the first module
+  /// procedure.
+  void cleanModuleEvaluationList() {
```

Comment:
Would `popModuleEvaluationList` be a better name choice?  (There are arguments against that one too, but I think it is more descriptive.)

Reply:
This is a "sync" review of code already in llvm.  So it may be best to avoid minor changes like this for now, but perhaps keep this in mind as a possible future update.

Reply:
I agree that `popModuleEvaluationList` is a better name. Will keep in mind.